testing/unclutter-xfixes/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
+# Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
+pkgname=unclutter-xfixes
+pkgver=1.2
+pkgrel=0
+pkgdesc="Hides mouse pointer while not in use"
+url="https://github.com/Airblader/unclutter-xfixes"
+arch="all"
+license="MIT"
+depends="libxfixes"
+makedepends="libxfixes-dev libev-dev libxi-dev asciidoc"
+install=""
+subpackages=""
+_myver="${pkgname}-${pkgver}"
+source="${_myver}.tar.gz::https://github.com/Airblader/${pkgname}/archive/v${pkgver}.tar.gz"
+builddir="${srcdir}/${pkgname}-${pkgver}"
+
+build() {
+	cd "${builddir}"
+	make || return 1
```

Comment:
The `|| return 1` isn't needed anymore since `abuild` after 3.5 automatically does `set -e`.